=== modified file 'bin/tools/config.py'
--- bin/tools/config.py	2011-05-17 05:54:43 +0000
+++ bin/tools/config.py	2012-06-18 12:08:56 +0000
@@ -168,7 +168,7 @@
         group.add_option("--test-file", dest="test_file", help="Launch a YML test file.")
         group.add_option("--test-report-directory", dest="test_report_directory", help="If set, will save sample of all reports in this directory.")
         group.add_option("--test-disable", action="store_true", dest="test_disable",
-                         default=False, help="Disable loading test files.")
+                         help="Disable loading test files.")
         group.add_option("--test-commit", action="store_true", dest="test_commit",
                          default=False, help="Commit database changes performed by tests.")
         parser.add_option_group(group)

