=== modified file 'bin/openerp-server.py'
--- bin/openerp-server.py	2011-01-06 13:06:54 +0000
+++ bin/openerp-server.py	2012-06-05 10:36:47 +0000
@@ -237,6 +237,20 @@
                 time.sleep(0.05)
     sys.exit(0)
 
+#----------------------------------------------------------
+# manage some platform specific behaviour
+#----------------------------------------------------------
+
+if sys.platform == 'win32':
+    import win32api
+    def mainthread_sleep():
+        # use SleepEx so the process can recieve console control event
+        # (required to Windows service survive if the user logout)
+        win32api.SleepEx(60000)
+else:
+    def mainthread_sleep():
+        time.sleep(60)
+
 if tools.config['pidfile']:
     fd = open(tools.config['pidfile'], 'w')
     pidtext = "%d" % (os.getpid())
@@ -248,7 +262,7 @@
 logger.info('OpenERP server is running, waiting for connections...')
 
 while netsvc.quit_signals_received == 0:
-    time.sleep(60)
+    mainthread_sleep()
 
 quit()
 

=== modified file 'win32/OpenERPServerService.py'
--- win32/OpenERPServerService.py	2010-12-29 11:51:44 +0000
+++ win32/OpenERPServerService.py	2012-06-05 10:22:41 +0000
@@ -87,9 +87,12 @@
         if not self.stopping:
             sys.exit("OpenERP Server check: server not running, check the logfile for more info")
 
-
+def ctrlHandler(ctrlType):
+    return True
 
 if __name__=='__main__':
+    # handle console control (used to survive to logout)
+    win32api.SetConsoleCtrlHandler(ctrlHandler, True)
     # Do with the service whatever option is passed in the command line
     win32serviceutil.HandleCommandLine(OpenERPServerService)
 

