=== modified file 'bin/tools/__init__.py'
--- bin/tools/__init__.py	2010-09-07 06:45:53 +0000
+++ bin/tools/__init__.py	2012-06-14 12:15:03 +0000
@@ -21,6 +21,7 @@
 
 import copy
 import win32
+import test_reports
 from config import *
 from misc import *
 from convert import *

=== modified file 'setup.py'
--- setup.py	2011-03-30 17:04:32 +0000
+++ setup.py	2012-06-14 12:14:33 +0000
@@ -53,12 +53,12 @@
             "dist_dir": 'dist',
             "packages": [
                 "lxml", "lxml.builder", "lxml._elementpath", "lxml.etree",
-                "lxml.objectify", "decimal", "xml", "xml", "xml.dom", "xml.xpath",
+                "lxml.objectify", "decimal", "xml", "xml", "xml.dom",
                 "encodings", "dateutil", "wizard", "pychart", "PIL", "pyparsing",
                 "pydot", "asyncore","asynchat", "reportlab", "vobject",
                 "HTMLParser", "select", "mako", "poplib",
                 "imaplib", "smtplib", "email", "yaml", "DAV",
-                "uuid", "commands",
+                "uuid", "commands", "mx.DateTime",
             ],
             "excludes" : ["Tkconstants","Tkinter","tcl"],
         }
@@ -144,6 +144,8 @@
         #for root, _, names in os.walk('pixmaps'):
         #    files.append((root, [join(root, name) for name in names]))
         files.append(('.', [join('bin', 'import_xml.rng'),]))
+        files.append(("Microsoft.VC90.CRT", glob.glob('C:\Microsoft.VC90.CRT\*.*')))
+        files.append((join('service','Microsoft.VC90.CRT'), glob.glob('C:\Microsoft.VC90.CRT\*.*')))
     else:
         man_directory = join('share', 'man')
         files.append((join(man_directory, 'man1'), ['man/openerp-server.1']))

