=== modified file 'bin/service/web_services.py'
--- bin/service/web_services.py	2012-07-11 14:26:46 +0000
+++ bin/service/web_services.py	2012-08-09 06:57:20 +0000
@@ -238,13 +238,14 @@
             tmpfile = (os.environ['TMP'] or 'C:\\') + os.tmpnam()
             file(tmpfile, 'wb').write(buf)
             args2=list(args2)
-            args2.append(' ' + tmpfile)
+            args2.append(tmpfile)
             args2=tuple(args2)
-        stdin, stdout = tools.exec_pg_command_pipe(*args2)
-        if not os.name == "nt":
+            res = tools.exec_pg_command(*args2)
+        else:
+            stdin, stdout = tools.exec_pg_command_pipe(*args2)
             stdin.write(base64.decodestring(data))
-        stdin.close()
-        res = stdout.close()
+            stdin.close()
+            res = stdout.close()
         if res:
             raise Exception, "Couldn't restore database"
         logger.notifyChannel("web-services", netsvc.LOG_INFO,

